poetry/packages/locker.py

```diff
@@ -595,3 +595,19 @@ def _dump_package(self, package: Package) -> dict:
                 data["develop"] = package.develop
 
         return data
+
+
+class NullLocker(Locker):
+    def __init__(self, locked=False):  # type: (bool) -> None
+        self._locked = locked
+
+    def is_locked(self):  # type: () -> bool
+        return self._locked
+
+    def set_lock_data(self, root, packages):  # type: (...) -> bool
+        return True
```

Comment:
What is the value of this method if it always `return True` and has no other side effects?

Reply:
The `NullLocker` was a leftover from the previous attempt at implementing the plugin system. I removed it.